nixos/modules/services/databases/redis.nix

```diff
@@ -87,8 +87,12 @@ in {
 
             port = mkOption {
               type = types.port;
-              default = 6379;
-              description = "The port for Redis to listen to.";
+              default = if name == "" then 6379 else 0;
+              defaultText = "6379 or 0 if name != \"\"";
```

Comment:
This description is a little confusing. Perhaps something like `6379 if name == "", otherwise 0` would be clearer?

Reply:
thanks, I've improved the message